configuration/etl/etl_action_defs.d/resource_specifications/resource_specs_aggregation.json

```
@@ -67,7 +67,7 @@
                 "name": "resource_allocated",
                 "alias": "ra",
                 "schema": "${SOURCE_SCHEMA}",
-                "on": "r.resource_id = ra.resource_id AND r.start_day_id = ra.start_day_id AND r.end_day_id = ra.end_day_id"
+                "on": "r.resource_id = ra.resource_id AND ra.start_day_id <= ${:PERIOD_END_DAY_ID} AND ra.end_day_id >= ${:PERIOD_START_DAY_ID}"
```

```suggestion
                "on": "r.resource_id = ra.resource_id AND ra.start_day_id <= ${:PERIOD_END_DAY_ID} AND ra.end_day_id >= ${:PERIOD_START_DAY_ID} AND ra.start_day_id <= r.end_day_id AND ra.end_day_id >= r.start_day_id"
```
otherwise it could join a `resource_allocated` that doesn't overlap temporally with the `resourcespecs`, e.g., in the diagram below, if the `|` represents a start/end boundary and time is on the x axis, we want `A` to join with `C` and `D` but not `E`.
```
                time  0   1   2   3   4
                      +---------------+
  aggregation period  |               |
                      +-------+-------+
       resourcespecs  |  A    |   B   |
                      +---+---+---+---+
  resource_allocated  | C |   D   | E |
                      +---+-------+---+
```